src/jobs_done10/server.py

```diff
@@ -73,6 +72,19 @@ def process_jobs_done(data) -> flask.Response:
          "refId": "refs/heads/stable-pwda11-master", "fromHash": "cd39f701ae0a729b73c57b7848fbd1f340a36514",
          "toHash": "8522b06a7c330008814a522d0342be9a997a1460", "type": "UPDATE"}]}
     """
+    if data is None:
```

Comment:
How does this "empty" request behaves with bitbucket "test webhook"?
I think there is a buttom for that in the bitbucket's webhook configuration.

Reply:
Sorry, I am not used to that page so I miss read the comment bellow.